### PR TITLE
Add NPM package badges 

### PR DESCRIPTION
**Pull request by AndrewDiMola**
Add package badges and a tl;dr section on where readers can find instructions on how to install the SDK for JS / React / Vue.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ The Text Editor SDK provides writing assistance to your end users across several
 - **Engagement** - Chooses the most compelling words and uses varied, engaging sentence structure.
 - **Delivery** - Helps end users find the right tone, eliminates hedging language (to sound more confident), finesses phrases to have more tact and empathy for readers, and supports respectful and current language practices.
 
+[Try the demo](https://developer.grammarly.com/docs/demo) to get a feel for the experience your users will have when you integrate Grammarly into your app.
+
+### Installing the SDK
+Add our real-time writing assistance into your app with just a few lines of code.
```
Comment (ljhaywar) [inline readme.md:17]:
```suggestion
Add our real-time writing assistance to your app with just a few lines of code. 
```

**Review by ljhaywar — APPROVED** [review]
Great call on linking to the npm packages!
giphy (12)

File: readme.md [hunk 2]
```diff
@@ -11,6 +11,28 @@ The Text Editor SDK provides writing assistance to your end users across several
...
+| [React]        | [![react-status]][react-package]  | Grammarly Text Editor SDK for React                     |
+| [Vue]          | [![vue-status]][vue-package]      | Grammarly Text Editor SDK for Vue 3 / Vue 2             |
+
+[JavaScript]: https://developer.grammarly.com/docs/editor-sdk-intro
+[React]: https://developer.grammarly.com/docs/editor-sdk-react
+[Vue]: https://developer.grammarly.com/docs/editor-sdk-vue
+
+[sdk-status]: https://img.shields.io/npm/v/@grammarly/editor-sdk.svg?style=flat&color=green&label=@grammarly/editor-sdk
+[sdk-package]: https://www.npmjs.com/package/@grammarly/editor-sdk
+[react-status]: https://img.shields.io/npm/v/@grammarly/editor-sdk-react.svg?style=flat&color=green&label=@grammarly/editor-sdk-react
```
[the remark this answers is not in the document]
Comment (AndrewDiMola) [reply readme.md:31]: Yes, the initial project has a CC0 license. 

TL;DR of CC0 is no restriction and donated to the public domain, including commercial use.

[the remark this answers is not in the document]
Comment (ljhaywar) [reply readme.md:31]: Thanks for verifying that!
